.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -0,0 +1,42 @@
+2.
+3.
+
+## Expected behaviour
+
+<!-- What did you expect to happen? -->
+
+## Actual behaviour
+
+<!-- What happened instead? Include relevant log lines (set ENABLE_HEALTHCHECK=false if logs are noisy). -->
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)